webpack.config.extensions.js

```diff
@@ -113,6 +113,10 @@ module.exports = [
 				..._.without( extensionsWebpackConfig.module.rules, transpileConfig ),
 			],
 		},
+		resolve: {
```

Comment:
cc @pento

Reply:
Originally added so that [this](https://github.com/Automattic/jetpack/pull/14075/commits/c3536a6ccb575b84f7ed76517a33afe4e6a31f40#diff-67c4a00b3bd3c01b617670a5fd26e417R13) worked. I haven't kept up with this PR: if it isn't including anything from `_inc/client` anymore, it can be reverted.